apps/site/src/content/friends/inox.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Inox Tools",
+  "logo": "https://inox-tools.fryuni.dev/_astro/shield.FOC531WK_Z23FUGT.webp",
```

Comment:
What goes into this hash? I haven't really looked into this part of the source code, but if something besides the original image is used to compute the hash it could change kinda at random. I think I should expose that as a stable URL 🤔.

Reply:
Do you have a stable URL for the image, doesn't have to come from the site, can also be a GitHub raw url?

Reply:
I mean that could be fine for this, we could even add `astro:assets` for remote images here 🤔